content/en/docs/ibm/create-cluster.md

```diff
@@ -132,6 +158,124 @@ Wait until the cluster is deployed and configured. It can take a while for the c
 
 ```shell
 ibmcloud ks clusters --provider ${WORKER_NODE_PROVIDER} |grep ${CLUSTER_NAME}|awk '{print "Name:"$1"\tState:"$3}'
+```
+## For VPC infrastructure.
+
+Here we are assuming, that a user starts from scratch and creates a new vpc instance,
+but realistically a user may have an existing VPC with a public gateway already attached and also a subnet to create a cluster.
+Please feel free to directly proceed to `Step 5` and substitute appropriate values. 
+
+* Step 1. Install a vpc-infrastructure plugin.
+
+`$ ibmcloud plugin install vpc-infrastructure`
+
+* Step 2. Target the gen 2 to access gen 2 resources.
+
+`$ ibmcloud is target --gen 2`
```

Comment:
Please use the following format (without indentation):

```
      ```shell
      ibmcloud etc etc
      ```

Reply:
Please don't use `$` in the beginning of each command line (like in the previous version of this guide)